src/autofill-utils.js

```diff
@@ -359,6 +361,7 @@ const getText = (el) => {
     if (el instanceof HTMLButtonElement) return removeExcessWhitespace(el.textContent)
 
     if (el instanceof HTMLInputElement && ['submit', 'button'].includes(el.type)) return el.value
+    if (el instanceof HTMLInputElement && el.type === 'image') return el.alt || el.value || el.title
```

Comment:
Do we need to call `removeExcessWhitespace` on these?

Reply:
Unlikely to include a lot of junk whitespace, but it doesn't hurt.